src/quo/components/inputs/recovery_phrase/view.cljs

```diff
@@ -60,7 +60,8 @@
         extra-props       (apply dissoc props custom-props)]
     [rn/view {:style (style/container container-style)}
      [rn/text-input
-      (merge {:accessibility-label    :recovery-phrase-input
+      (merge {:ref                    on-input-ref
+              :accessibility-label    :recovery-phrase-input
```

Comment:
I added support for accessing the text-input ref for the `recovery-phase-input` component. This was necessary since I wanted to have a way of showing the keyboard and error states after seed-phrase had been invalidated.

Currently, the error states for the recovery-phrase-input are only shown when the keyboard is shown, which led to using the text-input ref for focusing the input and automatically showing the keyboard.

Perhaps we don't need to use the text-input ref for showing the keyboard? Or maybe we can decide to always show the error states without the keyboard being shown. Thoughts?

Reply:
If the view is in a scroll view, we can use :keyboard-should-persist-taps`, otherwise your fix sounds good :+1: